VideoApi/VideoApi/Mappings/ParticipantToSummaryResponseMapper.cs

```diff
@@ -30,7 +30,7 @@ public static ParticipantSummaryResponse MapParticipantToSummary(ParticipantBase
                 DisplayName = participant.DisplayName,
                 Status = participantStatus.MapToContractEnum(),
                 UserRole = participant.UserRole.MapToContractEnum(),
-                HearingRole = participant is Participant ? ((Participant)participant).HearingRole : null,
+                HearingRole = participant.HearingRole, //participant is Participant ? ((Participant)participant).HearingRole : null,
```

Comment:
should the comment be removed, or is it left because we will have to revert this back, in which case should we point that out?

Reply:
Removed